synapse/rest/client/v2_alpha/room_keys.py

```diff
@@ -311,6 +312,7 @@ def __init__(self, hs):
         self.auth = hs.get_auth()
         self.e2e_room_keys_handler = hs.get_e2e_room_keys_handler()
 
+    @opentracing.trace_using_operation_name("get_room_keys_version")
```

Comment:
is this required, given we already start a span when the request arrives?

Reply:
Same as https://github.com/matrix-org/synapse/pull/5687#discussion_r310595716